docs/README.md

```diff
@@ -1,5 +1,13 @@
 # Argo Workflows
 
+[![slack](https://img.shields.io/badge/slack-argoproj-brightgreen.svg?logo=slack)](https://argoproj.github.io/community/join-slack)
+[![CI](https://github.com/argoproj/argo-workflows/workflows/CI/badge.svg)](https://github.com/argoproj/argo-workflows/actions?query=event%3Apush+branch%3Amaster)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3830/badge)](https://bestpractices.coreinfrastructure.org/projects/3830)
+[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/argo-workflows)](https://artifacthub.io/packages/helm/argo/argo-workflows)
```

Comment:
The officially supported manifests are the kustomize install.yaml, not the community maintained helm chart. I would hesitate putting this here so prominently since core maintainers are not actively ensuring these work / are up-to-date.

Reply:
Updated